### DISCUSSION TITLE
N64DD Roms Crash while loading

### DISCUSSION THREAD

**Discussion by Nilten1** [mupen64plus-ae/mupen64plus-ae#915] [Q&A]
Hi!
Sorry for my bad englisch im german.
So I wanted to play some N64DD Games on Android
but almost every game crashes while loading.
For example,  i load a Kyojin no Doshin (Doshin the Giant)
ROM, The N64 logo appears, The Title Screen loads and when i start the game the ROM freezes and i dont know why :/ Can someone help me please?

**Comment by fzurita** [comment]
Are you using the correct IPL version? You should be using the Japanese version. You can configure it under settings then data.

**Reply by Nilten1** [reply]
Yes i am using the IPL from Japan

**Reply by fzurita** [reply]
I just tried a bunch of 64DD games and most seem to be working ok. The only problem I'm seeing is Doshin the giant games are not booting up. Can you confirm that the only game not booting for you is that one?

**Reply by Nilten1** [reply]
SimCity 64 and Doshin the Giant Expansion Kit are not loading too. but the other games work fine.
